x-pack/plugins/apm/public/components/app/mobile/service_overview/stats/metric_item.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { Chart, Settings, Metric, MetricDatum } from '@elastic/charts';
+import { DARK_THEME } from '@elastic/charts';
+import { EuiLoadingContent, EuiPanel } from '@elastic/eui';
+
+export function MetricItem({
+  data,
+  id,
+  isLoading,
+}: {
+  data: MetricDatum[];
+  id: number;
+  isLoading: boolean;
+}) {
+  return (
+    <div
+      style={{
+        resize: 'none',
+        padding: '0px',
+        overflow: 'auto',
+        height: '100px',
+        borderRadius: '6px',
+      }}
+    >
+      {isLoading ? (
+        <EuiPanel hasBorder={true}>
+          <EuiLoadingContent lines={3} />
+        </EuiPanel>
+      ) : (
+        <Chart>
+          <Settings baseTheme={DARK_THEME} />
```

Comment:
Did you intend to use the dark theme always here?

Reply:
Initially, I started implementing the component to support dynamic coloring based on the comparison but didn't include it in this PR as a comparison is a stretch goal. Will remove the settings as it's not needed here